src/lgtm/install.sh

```
@@ -0,0 +1,29 @@
+bash "$SCRIPT_DIR/scripts/download/loki.sh" "${LOKIVERSION}"
+bash "$SCRIPT_DIR/scripts/download/grafana.sh" "${GRAFANAVERSION}" "${GRAFANAEDITION}"
+bash "$SCRIPT_DIR/scripts/download/tempo.sh" "${TEMPOVERSION}"
+bash "$SCRIPT_DIR/scripts/download/prometheus.sh" "${PROMETHEUSVERSION}"
+bash "$SCRIPT_DIR/scripts/download/pyroscope.sh" "${PYROSCOPEVERSION}"
+
+# Install
+
+
+# Configuration
+mkdir -p /etc/lgtm
+cp $SCRIPT_DIR/config/*.yaml /etc/lgtm/
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)